burn-import/src/burn/node/reshape.rs

```diff
@@ -0,0 +1,85 @@
+use super::{Node, NodeCodegen};
+use crate::burn::{Scope, TensorType, ToTokens, Type};
+use burn::record::PrecisionSettings;
+use burn::tensor::DataSerialize;
+use proc_macro2::TokenStream;
+use quote::quote;
+
+#[derive(Debug, Clone, new)]
+pub struct ReshapeNode {
+    pub input: TensorType,
+    pub output: TensorType,
+    pub shape: DataSerialize<i64>,
```

Comment:
You should use `Vec<usize>` here, `DataSerialize` is most appropriate for tensors, not shapes. Maybe you'll have to use the `convert_to_array` function.

Reply:
Since `Onnx` format uses `i64` values, wouldn't `Vec<i64>` be a better choice? `usize` depends too much on the architecture